sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c

```diff
@@ -33,6 +33,11 @@
 #include "debug.h"
 
 /**
+ * @brief Helper macro for LL_SEARCH to compare TCBs
+ */
+#define TCB_EQUAL(a,b)      ((a) == (b))
+
+/**
```

Comment:
at least for the use case here this should be safe, might be different when using the `LL_SORT*` macros, than `cmp` should work like `strcmp` return `-1`, `0`, or `1`; but that is not needed here.

Btw. here is the official [docu](http://troydhanson.github.io/uthash/), though it doesn't state differently on `cmp`, but for future references ....